features/step_definitions/web_steps.rb

```diff
@@ -76,10 +76,17 @@ def with_scope(locator)
 # TODO: Add support for checkbox, select or option
 # based on naming conventions.
 #
+# When /^(?:|I )fill in the following:$/ do |fields|
```

Comment:
Please Please do not edit this file. Everyone is referencing to this file, and we do not want to change everyone's code. We will eventually discuss about using this file at all, but please do not change this file